{"name": "test_register_user_with_invalid_data", "status": "broken", "statusDetails": {"message": "AttributeError: type object 'TestParaBank' has no attribute 'driver'", "trace": "self = <class 'paraBank_test.TestParaBank'>\n\n    def setup_class(self):\n>       self.pb = ParaBankPage(self.driver)\n                               ^^^^^^^^^^^\nE       AttributeError: type object 'TestParaBank' has no attribute 'driver'\n\ntestcases\\paraBank_test.py:19: AttributeError"}, "start": 1751089283196, "stop": 1751089283196, "uuid": "69b3d02f-c7c5-4b53-bcd2-2930954be469", "historyId": "c72fc3edbefed83b6e3076b54585fcc5", "testCaseId": "c72fc3edbefed83b6e3076b54585fcc5", "fullName": "testcases.paraBank_test.TestParaBank#test_register_user_with_invalid_data", "labels": [{"name": "parentSuite", "value": "testcases"}, {"name": "suite", "value": "paraBank_test"}, {"name": "subSuite", "value": "TestParaBank"}, {"name": "host", "value": "Hans"}, {"name": "thread", "value": "7720-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.paraBank_test"}]}